src/API.js

```diff
@@ -6,6 +6,12 @@ const checkForErrors = response => {
   }
 }
 
+export function getSingleUser(userNum) {
+  return fetch(`http://localhost:3001/api/v1/customers/${userNum}`)
+    .then(checkForErrors)
+    .catch(err => alert(err))
+}
+
```

Comment:
I did the same thing, nice work here. But just to be sure you're invoking a function with checkForErrors? Might want to throw parends on there to ensure that its calling properly